app/forms/provider_interface/pick_response_form.rb

```diff
@@ -15,6 +15,9 @@ def redirect_attrs
         attrs[:controller] = 'offer_changes'
         attrs[:action] = 'edit_offer'
         attrs[:step] = decision.delete_prefix('edit_')
+      elsif decision == 'new_reject'
```

Comment:
Assume this is necessary because we remove the feature flag dependant redirect?

Reply:
yea, I could redirect in the controller with
```
def new_reject
  redirect_to provider_interface_reasons_for_rejection_initial_questions_path(@application_choice)
end
```

but I though it may be better to do it here and avoid an extra step.

Reply:
:+1: I think this is the more obvious way to do it given the rest of the code uses the same method.